ChangeLog

```diff
@@ -41,6 +41,9 @@ Bugfix
    * Fix a potential integer overflow in the version verification for DER
      encoded X509 certificates. The overflow would enable maliciously
      constructed certificates to bypass the certificate verification check.
+   * Loop on length to seed, until all desired length is achieved, not to return
+     an error when the desired entropy length is higher than
+     MBEDTLS_ENTROPY_BLOCK_SIZE. Fix for #770.
```

Comment:
I think we should be a bit more specific in the ChangeLog, for example by mentioning the function that changed.

Reply:
ok